src/ensenso.cpp

```
@@ -283,8 +283,22 @@ void Ensenso::registerPointCloud() {
 	executeNx(command);
 }
 
+cv::Rect Ensenso::getRoi() {
```

Do we already use estd in dr_ensenso? If so, this should return a `Result<cv::Rect>`. Check other functions though, maybe they throw exceptions. In that case, let's throw an exception here too if the ROI is not found.

---

I assign an roi only if it exists or else i return an empty roi. Isnt that enough?

---

The caller doesn't know when the roi has been successfully constructed (you could say that roi.x/y/width/height == 0 is invalid, but returning `Result<cv::Rect>` is more clear / verbose on what is happening).